cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server_data_adapters.go

```diff
@@ -134,3 +136,51 @@ func (s *Server) getAvailablePackageDetail(pkgMetadata *datapackagingv1alpha1.Pa
 
 	return availablePackageDetail, nil
 }
+
+func (s *Server) getInstalledPackageSummary(pkgInstall *packagingv1alpha1.PackageInstall, pkgMetadata *datapackagingv1alpha1.PackageMetadata, pkgVersionsMap map[string][]pkgSemver, cluster string) (*corev1.InstalledPackageSummary, error) {
```

Comment:
I'd normally reserve the `get` prefix for a function if it is getting/fetching something... ie something that is network constrained. What you're doing here is creating an installed package summary based on data that's already been fetched (ie. no network here). Maybe `installedPackageSummaryFromData` or something without `get` would be clearer.

Reply:
Yep, I'm with the Java folks who like using getters, but sounds fair :P  However, I feel like I do need a verb for a function name, so... what about `buildInstalledPackageSummary(...` ?